pytest_tank_test/__init__.py

```diff
@@ -55,6 +55,13 @@ def pytest_configure(config):
     try:
         repo = Repository(cur_dir)
     except RuntimeError:
+        valid_repo = False
+    else:
+        # Make sure we're in a toolkit component
+        valid_repo = repo.is_toolkit_component()
```

Comment:
Your fix broke the all tests actually. A valid repo is a repo that does not raise. You can simply set True in the else and you're good.

Reply:
Please fix it and the surrounding comments.

Reply:
Sorry about that, I had a chance to take a look just now and I think I've addressed the failing tests.

However, I believe as of this [commit](https://github.com/shotgunsoftware/tk-toolchain/commit/ee1043c5d3dfd1016b9739ef42e6773b8f365e95) Repo init will not raise any longer.  This was an attempt to re-add that behavior, but it looks like I missed a couple possibilities when adding the validity check.  I think my latest commit addresses that and restores the expected behavior... let me know!